spring-integration-core/src/main/java/org/springframework/integration/transaction/PassthroughTransactionSynchronizationFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.transaction;
+
+import org.springframework.transaction.support.TransactionSynchronization;
+import org.springframework.transaction.support.TransactionSynchronizationManager;
+import org.springframework.util.Assert;
+
+/**
+ * Stub implementation of {@link TransactionSynchronizationFactory} which allows you to create a
```

Comment:
M-m-m. I don't like "stub" word. Let's say "A simple TransactionSynchronizationFactory implementation which creates blah-blah and register blah-blah"

"Register" is very important word here because we do `TransactionSynchronizationManager.bindResource()` unconditionally.